scripts/build/configure/termux_step_configure.sh

```diff
@@ -2,10 +2,42 @@ termux_step_configure() {
 	[ "$TERMUX_PKG_METAPACKAGE" = "true" ] && return
 
 	if [ "$TERMUX_PKG_FORCE_CMAKE" = "false" ] && [ -f "$TERMUX_PKG_SRCDIR/configure" ]; then
+		if [ "$TERMUX_CONTINUE_BUILD" == "true" ]; then
+			return;
+		fi
 		termux_step_configure_autotools
-	elif [ -f "$TERMUX_PKG_SRCDIR/CMakeLists.txt" ]; then
-		termux_step_configure_cmake
+	elif [ "$TERMUX_PKG_FORCE_CMAKE" = "true" ] || [ -f "$TERMUX_PKG_SRCDIR/CMakeLists.txt" ]; then
+		if [ "$TERMUX_CONTINUE_BUILD" == "true" ]; then
+			termux_setup_cmake
+			if [ "$TERMUX_CMAKE_BUILD" = Ninja ]; then
+				termux_setup_ninja
+			fi
+			return;
+		fi
+		termux_setup_cmake
+		if [ "$TERMUX_CMAKE_BUILD" = Ninja ]; then
+			termux_setup_ninja
+			MAKE_PROGRAM_PATH=$(command -v ninja)
```

Comment:
You can move all these lines back to `termux_step_configure_cmake`, as they're only used there. You only want `termux_setup_{cmake,ninja}` moved up here.